marimo/_ast/parse.py

```diff
@@ -187,7 +188,9 @@ def to_cell_def(
         if node.lineno - node.body[0].lineno == 0:
             # Quirk where the ellipse token seems to have a line index at
             # the end of the dots ...<
-            if isinstance(getattr(node.body[0], "value", None), ast.Ellipsis):
+            if sys.version_info < (3, 14) and isinstance(
+                getattr(node.body[0], "value", None), ast.Ellipsis
+            ):
```

Comment:
Don't we still need to handle ellipses in 3.14? I thought the `ast` API changed, but Ellipses still exist